ui/component/or-asset-viewer/src/or-edit-asset-panel.ts

```diff
@@ -204,7 +204,15 @@ export class OrEditAssetPanel extends LitElement {
 
     shouldUpdate(changedProperties: PropertyValues): boolean {
         if (changedProperties.has("asset")) {
-            this.changedAttributes = [];
+            const oldAsset = changedProperties.get("asset") as Asset;
+            if (this.asset.attributes && oldAsset?.attributes) {
```

Comment:
We should probably test for the asset ID being the same as well and if it isn't then all attributes have changed